lib/internal/per_context/domexception.js

```diff
@@ -71,16 +105,34 @@ class DOMException extends Error {
     if (internals === undefined) {
       throw new ERR_INVALID_THIS('DOMException');
     }
-    const code = nameToCodeMap.get(internals.name);
-    return code === undefined ? 0 : code;
+    return internals.code;
+  }
+
+  [kClone]() {
+    const message = this.message;
+    const name = this.name;
+    const code = this.code;
+    return {
+      data: { message, name, code },
+      deserializeInfo: 'internal/per_context/domexception:DOMException',
+    };
+  }
+
+  [kDeserialize]({ message, name, code }) {
+    ensureInitialized();
+    internalsMap.set(this, { message: `${message}`, name: `${name}`, code });
   }
 }
 
 ObjectDefineProperties(DOMException.prototype, {
-  [SymbolToStringTag]: { configurable: true, value: 'DOMException' },
-  name: { enumerable: true, configurable: true },
-  message: { enumerable: true, configurable: true },
-  code: { enumerable: true, configurable: true }
+  [SymbolToStringTag]: { configurable: true, enumerable: false,
+                         value: 'DOMException', writable: false },
+  name: { configurable: true, enumerable: true,
+          value: undefined, writable: false },
+  message: { configurable: true, enumerable: true,
+             value: undefined, writable: false },
+  code: { configurable: true, enumerable: true,
+          value: undefined, writable: false }
```

Comment:
```suggestion
  [SymbolToStringTag]: {
    configurable: true,
    enumerable: false,
    value: 'DOMException',
    writable: false,
  },
  name: {
    configurable: true,
    enumerable: true,
    value: undefined,
    writable: false,
  },
  message: {
    configurable: true,
    enumerable: true,
    value: undefined,
    writable: false,
  },
  code: {
    configurable: true,
    enumerable: true,
    value: undefined,
    writable: false,
  }
```

Reply:
done in https://github.com/nodejs/node/pull/41148/commits/c5c7858e7b91da39e4defcbc11e9506159e3ae84